src/Executables/Game/PacketHandlers/PongHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Extensions.Logging;
+using QuantumCore.API;
+using QuantumCore.API.PluginTypes;
+using QuantumCore.Caching;
+using QuantumCore.Core.Event;
+using QuantumCore.Game.Packets;
+using Ack = QuantumCore.Game.Packets.Ping;
```

Comment:
Why a custom name here?

Reply:
This ping packet won't be responded to with a pong by the client; it's simply an acknowledgment packet. I thought assigning an alias to the packet might make the code more readable and prevent confusion.

Reply:
Well if that's the case then it's ok. However I would like to see a comment about that in the code because it's not easy to understand?